modules/specifying-infrastructure-vsphere-failure-domains.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/sphere-failure-domain-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="specifying-infrastructure-vsphere-failure-domains_{context}"]
+= Specifying multiple failure domains for your cluster on VSphere
+
+You can configure the `infrastructures.config.openshift.io` configuration resource to specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance.
+////
+OCP cluster or VMware vSphere cluster
+////
+
+Topology-aware features for the cloud controller manager and the vSphere Container Storage Interface (CSI) Operator Driver require information about the vSphere topology where you host your {product-title} cluster. This topology information exists in the `infrastructures.config.openshift.io` configuration resource. 
+
+Before you specify failure domains for your cluster, you must ensure that all datacenters and compute clusters contain tags, so that the cloud provider can add labels to your node. For example, if `datacenter-1` represents `region-a` and `compute-cluster-1` represents `zone-1`, the cloud provider adds an `openshift-region` category label with a value of `region-a` to `datacenter-1`.  Additionally, the cloud provider adds an `openshift-zone` category tag with a value of `zone-1` to `compute-cluster-1`.
+
+[NOTE]
+====
+You can migrate control plane nodes with vMotion capabilities to a failure domain. After you add these nodes to a failure domain, the cloud provider adds `topology.kubernetes.io/zone` and `topology.kubernetes.io/region` labels to these nodes.  
+====
+
+.Prerequisites
+* You created the `openshift-region` and `openshift-zone` tag categories on the vCenter server.
+* You ensured that all failure domains share a common layer 3 network. This configuration ensures that API and Ingress VirtualIP (VIP) addresses can interact with your cluster.
+* You ensured that all datacenters and compute clusters contain tags that represent the name of their associated region or zone, or both. l
+
+[IMPORTANT]
+====
+If you do not supply tags to all datacenters and compute classes before you create a node or migrate a node, the cloud provider cannot add the `topology.kubernetes.io/zone` and `topology.kubernetes.io/region` labels to the node. This means that a service cannot route traffic to your node.
+====
+
+.Procedure
+
+. Edit the `infrastructures.config.openshift.io` custom resource definition (CRD) of your cluster to specify multiple failure domains in the `failureDomains` section of the resource:
++
+[source,terminal]
+----
+$ oc edit infrastructures.config.openshift.io cluster
+----
++
+.Example `infrastructures.config.openshift.io` CRD for a instance named `cluster` with multiple failure domains defined in its configuration.
++
+[source,yaml]
+----
+spec:
+  cloudConfig:
```

Comment:
HI @mjpytlak . Just want to get your opinion on adding a list of items under a procedure step. Would adding something similar (if not almost identical) to the definition list outlined in https://github.com/openshift/openshift-docs/pull/55767/files be OK in a procedural step? 

I think providing a link might cause confusion, as a user would be jumping from a post-intall config to a pre-install config. Maybe avoiding duplication of content would be a better approach and a link would suffice?